### DISCUSSION TITLE
How can I change the color of pointer while hovering over the Area graph ?

### DISCUSSION THREAD

**Discussion by ProPhycient5** [ant-design/ant-design-charts#2138] [Q&A] [answered]
Please respond, I have attached the code and official link of that example.

https://ant-design-charts.antgroup.com/en/examples/area/basic/#basic-gradients

```
const AreaGraph = ({ transformedData }) => {
  const config = {
    data: transformedData,
    xField: "x_value",
    yField: "value",
    xAxis: {
      range: [0, 1],
      tickCount: 12,
    },
    yAxis: {
      grid: {
        line: {
          style: {
            stroke: "#303030",
            lineWidth: 1.5,
            cursor: "pointer",
          },
        },
      },
    },
    areaStyle: () => {
      return {
        fill: "l(90) 0:#ff6708 0.5:#ff6403 0.75:#bd4c06 1:#0d0500",
      };
    },
    line: {
      style: {
        stroke: "#E36B20",
      },
    },
    tooltip: {
      customContent: (title, items) => {
        return <CustomTooltip payload={items} label={title} />;
      },
    },
  } as any;

  return <Area {...config} style={{ height: "98%", width: "98%" }} />;
};
```

**Comment by ChenKun1997** [comment]
@ProPhycient5  Is this what you want ?
![image](https://github.com/ant-design/ant-design-charts/assets/24249578/b23ff0ac-4d31-47e6-a9be-2ec504961d66)

**Reply by ProPhycient5** [reply]
Thanks @ChenKun1997 for your response. No I am asking motionary vertical line that appears when we hover over the graph, how can I hide that. 
The attached GIF shows a vertical bar that appears, either there should be a way to make it transparent or hide it simply.

![area_graph_pointer](https://github.com/ant-design/ant-design-charts/assets/71059909/31e21b8f-429d-4257-823e-0a41ce14f0ec)

**Reply by ChenKun1997** [reply, marked as answer]
@ProPhycient5 
![image](https://github.com/ant-design/ant-design-charts/assets/24249578/32a3aae5-80f2-4f21-a55c-74608dfbb928)

**Reply by ProPhycient5** [reply]
Thanks @ChenKun1997, it worked. Nameste 🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻🙏🏻
